src/backend/distributed/shared_library_init.c

```diff
@@ -380,18 +379,6 @@ RegisterCitusConfigVariables(void)
 		GUC_UNIT_MS | GUC_STANDARD,
 		NULL, NULL, NULL);
 
-	/* keeping temporarily for updates from pre-6.0 versions */
```

Comment:
I'm not the best person to ask, but I think we prefer not to remove GUCs on minor release versions. @metdos is that correct?

Reply:
I think this GUC has had enough time to no longer be used.